pageant_windows.go

```diff
@@ -17,6 +17,7 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 //
 
+//go:build windows
```

Comment:
Guess we should also add a Go 1.17 style build tag here to make it complete? https://github.com/xanzy/ssh-agent/blob/master/sshagent.go#L17

Reply:
Fixed!